README.md

```diff
@@ -61,7 +61,7 @@ MetricFlow is distributed under a Business Source License (BUSL-1.1). For detail
 MetricFlow can be installed from PyPi for use as a Python library with the following command:
 
 ```
-pip install metricflow
```

Comment:
I think this is good for now. I'm going to need to do a broader overhaul on this file once I finish forking out all of the SqlClient implementations, but it's better to get rid of instructions to install metricflow sooner rather than later.

Reply:
thank you so very much @tlento - much appreciated!